src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/HelpMessage.java

```diff
@@ -39,6 +39,11 @@ final public class HelpMessage {
             "                  NVRs). The syntax is described below\n" +
             "    --history <number>\n" +
             "                  To specify the maximum number of builds to look in.\n" +
+            "    --exact-length <number>\n" +
```

Comment:
tis exaplins a bit... The help hsoudl always should be part of the commit adding the switch. Anfother +1 for clas CmdlineArgument O:)